tools/web3-packagetemplate/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "web3-packagetemplate",
-	"version": "0.1.1-alpha.0",
+	"version": "0.1.1-alpha.1",
```

Comment:
I think there isn't any change in this package ?, so we dnt need version bump.  Its easier to review if we include changelog changes in same PR.